pkg/platform/yaml/mockdb/db.go

```diff
@@ -326,3 +326,73 @@ func (ys *MockYaml) filterByTimeStamp(_ context.Context, m []*models.Mock, after
 	}
 	return filteredMocks, unfilteredMocks
 }
+func (ys *MockYaml) GetHTTPMocks(ctx context.Context, testSetID string, mockPath string) ([]*models.HTTPSchema2, error) {
```

Comment:
Since all HTTP mocks are unfiltered mocks, you call that function and filter on the basis of the mock kind. (HTTP)